ios/keyman/Keyman/Keyman/SetUpViewController/SetUpViewController.swift

```diff
@@ -62,16 +65,16 @@ class SetUpViewController: UIViewController, UIWebViewDelegate {
 
     // Yes, .php.html.  That's how `wget` is set to retrieve it, since Safari won't recognize the contents
     // without the .html ending, it seems.
-    let filePath = offlineHelpBundle.path(forResource: "installing-system-keyboard.php",
+    let filePath = offlineHelpBundle.path(forResource: "installing-system-keyboard",
                                           ofType: "html",
-                                          inDirectory: nil)
+                                          inDirectory: "start")
     webView.loadRequest(URLRequest(url: URL.init(fileURLWithPath: filePath!)))
   }
 
   private func loadFromServer() {
     let appVersion = Version.current.majorMinor
     let url = "https://help.keyman.com/products/iphone-and-ipad/\(appVersion.plainString)"
-      + "/installing-system-keyboard.php?embed=ios"
+      + "/start/installing-system-keyboard"
```

Comment:
Even if we aren't currently using the `embed` param, I'd like to keep it in so that we can use it in the future on the site, if we are going to use the online help from the app.